test/unit/lib/plugins/aws/package/compile/events/sqs.test.js

```diff
@@ -676,3 +676,39 @@ describe('AwsCompileSQSEvents #2', () => {
     expect(eventSourceMappingResource.DependsOn).to.deep.equal([]);
   });
 });
+
+describe('when functionResponseTypes property is given', () => {
```

Comment:
Let's rename title to `regular configuration` (ideally this should be run, in which we will test all properties, now starting with `functionResponseType`